### DISCUSSION TITLE
Read HomePod sensor values

### DISCUSSION THREAD

**Discussion by flohse123** [HomeSpan/HomeSpan#520] [Q&A] [answered]
With iOS 16.3 Apple has activated the temp/humidity sensor in HomePod mini and HomePod 2. The values are shown within the home app. Is it possible to read these values with HomeSpan in order to use them in the logic of other accessories?

**Comment by HomeSpan** [comment, marked as answer]
I'm unaware of any method by which one Accessory can read the information from another Accessory directly through HomeKit, though if you create your own Accessories you can create non-HomeKit modes of communication (i.e. ESP-NOW for ESP-32 devices).   Within iOS you can of course read the info from any device that is on your HomeKit network, either via the Home App, Eve, etc., and set up automations that trigger one device to act based on the status of another device.
